ddtrace/contrib/django/utils.py

```diff
@@ -121,16 +120,9 @@ def get_request_uri(request):
                 return None
         urlparts[k] = v
 
-    # DEV: With PY3 urlunparse calls urllib.parse._coerce_args which uses the
-    # type of the scheme to check the type to expect from all url parts, raising
-    # a TypeError otherwise. If the scheme is not a str, the function returns
-    # the url parts bytes decoded along with a function to encode the result of
-    # combining the url parts. We returns a byte string when all url parts are
-    # byte strings.
-    # https://github.com/python/cpython/blob/02d126aa09d96d03dcf9c5b51c858ce5ef386601/Lib/urllib/parse.py#L111-L125
-    if PY3 and not all(isinstance(value, binary_type) or value is None for value in urlparts.values()):
+    if PY3:
```

Comment:
@majorgreys do we need this logic anymore?

Reply:
We can coerce these values as we do elsewhere by using `six.ensure_text`, which helps us avoid having to handle the version differences.